blocks/share-bar-block/README.md

```diff
@@ -25,26 +34,21 @@ Outline any schema information requirements necessary to know for ths block
 
 ## Internationalization fields
 
-| Phrase key | Default (English)     |
-| ---------- | --------------------- |
-| `key`      | `english translation` |
+| Phrase key              | Default (English)                           |
+| ----------------------- | ------------------------------------------- |
+| share-button-aria-label | "Share current article via `%{socialType}`" |
+
+**Note:** The `%{socialType}` value is taken from Internationalization fields in the
+[global-phrases-block](https://github.com/WPMedia/arc-themes-blocks/tree/arc-themes-release-version-2.0.1/blocks/global-phrases-block).
 
 ## Events
```

Comment:
No need to document Events and Event Listening - this block does not emit any events. This documentation is for when a block is emitting custom events

Reply:
Gotcha! Updated. Thanks!